src/main/java/supercoder79/simplexterrain/mixin/MixinLevelGeneratorTypeServer.java

```diff
@@ -1,23 +0,0 @@
-package supercoder79.simplexterrain.mixin;
-
-import org.spongepowered.asm.mixin.Final;
-import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
-import org.spongepowered.asm.mixin.injection.At;
-import org.spongepowered.asm.mixin.injection.Inject;
-import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
-
-import net.minecraft.world.level.LevelGeneratorType;
-import supercoder79.simplexterrain.world.WorldType;
-
-@Mixin(LevelGeneratorType.class)
-public class MixinLevelGeneratorTypeServer {
```

Comment:
this is the mixin that is still neccesary

Reply:
it isn't, as level-type getting is moved in api to world gen stage

Reply:
Yeah I realised

Reply:
I took a look at your pr and saw you had some mixins to server properties